data/avro-schema/src/main/resources/avro/net/corda/data/uniqueness/UniquenessCheckExternalResponse.avsc

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Is there a reason this file was added / deleted instead of moved like the others? Moving preserves change history.

Reply:
I have no idea tbh, I did the same thing as with the other files but for some reason it was deleted/recreated. Maybe some IntelliJ issue?